File: Backend/Models/Documento.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Backend.Models
{
    [Table("Documentos")]
    public class Documento
    {
        [Key]
        public int Id { get; set; }

        [StringLength(14)]
        public string? CPF { get; set; }

        public string? RG { get; set; }

        public string? DtEmissaoRG { get; set; }

        public string? OrgaoExpeditor { get; set; }

        public int? UF_RG_IdEstado { get; set; } 

        [ForeignKey("UF_RG_IdEstado")] 
        public Estado? UF_RG_Estado { get; set; } 

        public string? CTPS { get; set; }

        public string? SerieCTPS { get; set; }

        public string? DtEmissaoCTPS { get; set; }

        public int? UF_CTPS_IdEstado { get; set; } 

        [ForeignKey("UF_CTPS_IdEstado")] 
        public Estado? UF_CTPS_Estado { get; set; } 

        [StringLength(14)]
        public string? PIS { get; set; }
    }
}

File: Frontend/src/app/components/interface/documento.ts
export interface Documento {
    Id: number;
    CPF: string;
    RG: number;
    DtEmissaoRG?: string | null | undefined;
    OrgaoExpeditor?: string | null | undefined;
    UF_RG_IdEstado?: string | null | undefined;
    CTPS?: number | null | undefined;
    SerieCTPS?: number | null | undefined;
    DtEmissaoCTPS?: string | null | undefined;
    UF_CTPS_IdEstado?: string | null | undefined;
    PIS?: string;
}